docker/build-machine.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright 2023 Cartesi Pte. Ltd.
+# Copyright 2022 Cartesi Pte. Ltd.
```

Comment:
@fmoura in fact please be aware of the thread discussing the copyright header format to apply to the files in the Honeypot. https://discord.com/channels/600597137524391947/1130905237901803560